Polyhedron/demo/Polyhedron/Polyhedron_type.h

```diff
@@ -193,10 +195,38 @@ class Polyhedron_demo_items : public CGAL::Polyhedron_items_3 {
 };
 
 #include "Polyhedron_type_fwd.h"
+#include <CGAL/Polygon_mesh_processing/properties.h>
+inline
+boost::property_map<Polyhedron, boost::vertex_index_t>::type
+get(CGAL::vertex_selection_t, const Polyhedron& p)
+{
+  return get(boost::vertex_index,p);
+}
+inline
+boost::property_map<Polyhedron, boost::face_index_t>::type
+get(CGAL::face_selection_t, const Polyhedron& p)
+{
+  return get(boost::face_index,p);
+}
+
+namespace boost
+{
+template <>
+struct property_map<Polyhedron, CGAL::vertex_selection_t>
+{
+  typedef boost::property_map<Polyhedron,boost::vertex_index_t>::type type;
+  typedef boost::property_map<Polyhedron,boost::vertex_index_t>::const_type const_type;
+};
+template <>
+struct property_map<Polyhedron, CGAL::face_selection_t>
+{
+  typedef boost::property_map<Polyhedron,boost::face_index_t>::type type;
+  typedef boost::property_map<Polyhedron,boost::face_index_t>::const_type const_type;
+};
+}//boost
```

Comment:
What are the lines 198-266 supposed to do? Why are they in a private header of the demo, and not in the CGAL headers?